style.css

```diff
@@ -1,1160 +1,1151 @@
 * {
-    padding: 0;
-    margin: 0;
-    box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
 }
 
 body {
-    font-family: 'Roboto', Arial, Helvetica, sans-serif;
-    justify-content: center;
-    align-items: center;
-    box-sizing: border-box;
-    margin: 0;
-    padding: 0;
+  font-family: 'Roboto', Arial, Helvetica, sans-serif;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
 }
 
 header {
-    width: auto;
-    background: rgba(62, 60, 60, 1);
-    height: 92px;
-    margin: 0;
-    display: flex;
-    align-items: center;
-    list-style: none;
+  width: auto;
+  background: rgba(62, 60, 60, 1);
+  height: 92px;
+  margin: 0;
+  display: flex;
+  align-items: center;
+  list-style: none;
 }
 
 .mylogo {
-    list-style: none;
-    padding-left: 2rem;
+  list-style: none;
+  padding-left: 2rem;
 }
 
 .hamburger {
-    z-index: 100;
-    top: 1rem;
-    padding: 4px;
-    cursor: pointer;
-    margin-left: 13rem;
+  z-index: 100;
+  top: 1rem;
+  padding: 4px;
+  cursor: pointer;
+  margin-left: 13rem;
 }
```

Comment:
- [OPTIONAL] The cross button is behaving as expected on `375px` However, when I am increasing the width, the button is misplaced ⚠😓 please make sure that it is always placed at the end, could you please fix this? ✨

![image](https://github.com/richie1988/Microverse-websitepagesproject/assets/106119105/3e2ec91a-de6f-49fb-a571-1ec2f2d5664f)